### PR TITLE
KeyframeEffect: added more code for better understanding of the context

### PR DESCRIPTION
**Pull request by OnkarRuikar**
https://developer.mozilla.org/en-US/docs/Web/API/KeyframeEffect#examples

## Summary
Added more code to the example for better understanding of the context in which the interface is used.

As per the BCD table the interface is not experimental anymore.

#### Motivation
Just creating an object was not giving enough understanding of it's use in the Example section.

#### Metadata

- [x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/api/keyframeeffect/index.md [hunk 1]
```diff
@@ -55,14 +54,21 @@ _This interface inherits some of its methods from its parent, {{domxref("Animati
...
       { transform: 'translateY(0%)' }, // keyframe
       { transform: 'translateY(100%)' } // keyframe
     ],
     { duration: 3000, fill: 'forwards' } // keyframe options
   );
+  
+  let rabbitDownAnimation = new Animation(rabbitDownKeyframes, document.timeline);
+  
+  // Play rabbit animation
+  rabbitDownAnimation.play();
```
Comment (wbamberg) [inline files/en-us/web/api/keyframeeffect/index.md:71]: The code in this example is indented.

File: files/en-us/web/api/keyframeeffect/index.md [hunk 2]
```diff
@@ -55,14 +54,21 @@ _This interface inherits some of its methods from its parent, {{domxref("Animati
 In the [Follow the White Rabbit example](http://codepen.io/rachelnabors/pen/eJyWzm/?editors=0010), the KeyframeEffect constructor is used to create a set of keyframes that dictate how the White Rabbit should animate down the hole:
 
 ```js
- var rabbitDownKeyframes = new KeyframeEffect(
+ let whiteRabbit = document.getElementById("rabbit");
+
+ let rabbitDownKeyframes = new KeyframeEffect(
```
Comment (wbamberg) [inline files/en-us/web/api/keyframeeffect/index.md:59]:
```suggestion
const rabbitDownKeyframes = new KeyframeEffect(
```

File: files/en-us/web/api/keyframeeffect/index.md [hunk 3]
```diff
@@ -55,14 +54,21 @@ _This interface inherits some of its methods from its parent, {{domxref("Animati
 In the [Follow the White Rabbit example](http://codepen.io/rachelnabors/pen/eJyWzm/?editors=0010), the KeyframeEffect constructor is used to create a set of keyframes that dictate how the White Rabbit should animate down the hole:
 
 ```js
- var rabbitDownKeyframes = new KeyframeEffect(
+ let whiteRabbit = document.getElementById("rabbit");
```
Comment (wbamberg) [inline files/en-us/web/api/keyframeeffect/index.md:57]:
```suggestion
const whiteRabbit = document.getElementById('rabbit');
```

File: files/en-us/web/api/keyframeeffect/index.md [hunk 4]
```diff
@@ -55,14 +54,21 @@ _This interface inherits some of its methods from its parent, {{domxref("Animati
...
+ let rabbitDownKeyframes = new KeyframeEffect(
     whiteRabbit, // element to animate
     [
       { transform: 'translateY(0%)' }, // keyframe
       { transform: 'translateY(100%)' } // keyframe
     ],
     { duration: 3000, fill: 'forwards' } // keyframe options
   );
+  
+  let rabbitDownAnimation = new Animation(rabbitDownKeyframes, document.timeline);
```
Comment (wbamberg) [inline files/en-us/web/api/keyframeeffect/index.md:68]:
```suggestion
const rabbitDownAnimation = new Animation(rabbitDownKeyframes, document.timeline);
```

File: files/en-us/web/api/keyframeeffect/keyframeeffect/index.md [hunk 5]
```diff
@@ -82,14 +81,21 @@ The single argument constructor (see above) creates a clone of an existing  {{do
 In the [Follow the White Rabbit example](https://codepen.io/rachelnabors/pen/eJyWzm/?editors=0010), the `KeyframeEffect` constructor is used to create a set of keyframes that dictate how the White Rabbit should animate down the hole:
 
 ```js
-var rabbitDownKeyframes = new KeyframeEffect(
+let whiteRabbit = document.getElementById("rabbit");
```
Comment (wbamberg) [inline files/en-us/web/api/keyframeeffect/keyframeeffect/index.md:84]:
```suggestion
const whiteRabbit = document.getElementById('rabbit');
```

File: files/en-us/web/api/keyframeeffect/keyframeeffect/index.md [hunk 6]
```diff
@@ -82,14 +81,21 @@ The single argument constructor (see above) creates a clone of an existing  {{do
...
+
+let rabbitDownKeyframes = new KeyframeEffect(
```
Comment (wbamberg) [inline files/en-us/web/api/keyframeeffect/keyframeeffect/index.md:86]:
```suggestion
const rabbitDownKeyframes = new KeyframeEffect(
```

File: files/en-us/web/api/keyframeeffect/keyframeeffect/index.md [hunk 7]
```diff
@@ -82,14 +81,21 @@ The single argument constructor (see above) creates a clone of an existing  {{do
...
     whiteRabbit, // element to animate
     [
       { transform: 'translateY(0%)' }, // keyframe
       { transform: 'translateY(100%)' } // keyframe
     ],
     { duration: 3000, fill: 'forwards' } // keyframe options
   );
+  
+let rabbitDownAnimation = new Animation(rabbitDownKeyframes, document.timeline);
```
Comment (wbamberg) [inline files/en-us/web/api/keyframeeffect/keyframeeffect/index.md:95]:
```suggestion
const rabbitDownAnimation = new Animation(rabbitDownKeyframes, document.timeline);
```

**Review by wbamberg — CHANGES_REQUESTED** [review]
Thank you @OnkarRuikar ! It would be great (IMO) to make these into live samples, but this is already much better. I had a few little comments.

**Comment by OnkarRuikar** [comment]
@wbamberg  tanks for the suggestions! I've committed the changes.

File: files/en-us/web/api/keyframeeffect/index.md [hunk 8]
```diff
@@ -55,14 +54,21 @@ _This interface inherits some of its methods from its parent, {{domxref("Animati
...
       { transform: 'translateY(0%)' }, // keyframe
       { transform: 'translateY(100%)' } // keyframe
     ],
     { duration: 3000, fill: 'forwards' } // keyframe options
   );
+  
+const rabbitDownAnimation = new Animation(rabbitDownKeyframes, document.timeline);
+  
+  // Play rabbit animation
+  rabbitDownAnimation.play();
```
Comment (wbamberg) [inline files/en-us/web/api/keyframeeffect/index.md:71]:
```suggestion
// Play rabbit animation
rabbitDownAnimation.play();
```

**Review by wbamberg — APPROVED** [review]
👍 thank you @OnkarRuikar !

**Pull request merged**